components/engine_rocks/src/engine.rs

```diff
@@ -108,47 +80,31 @@ impl KvEngine for Rocks {
     fn cf_names(&self) -> Vec<&str> {
         self.0.cf_names()
     }
-
-    fn delete_all_in_range_cf(
```

Comment:
why removing this?

Reply:
From the commit message:

> The code is unused, is duplicated between engine_rocks and engine and is not the
    same. I'm going to come back to this when it actually needs to be used.

I'm suspicious of it.

Reply:
https://github.com/tikv/tikv/blob/master/src/raftstore/store/fsm/apply.rs#L1293 and https://github.com/tikv/tikv/blob/master/src/server/gc_worker.rs#L391 use delete_all_in_range_cf.